### ISSUE TITLE
[Feature]: Moon Phase Calculator

### ISSUE THREAD

**Issue by aasthakourav20** [Rakesh9100/CalcDiverse#1172]
### Description

I would like to add a Moon Phase Calculator. Kindly assign it to me under GSSoC'24

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a GSSOC'24 contributor.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by Rakesh9100** [comment]
Do you want to be assigned @aasthakourav20

**Comment by aasthakourav20** [comment]
> Do you want to be assigned @aasthakourav20

Yes , I want to  create an Application that provides information on moon phases and weather .Made with HTML, CSS and Javascript.

**Comment by Rakesh9100** [comment]
What input options will you give, will you consider Country selection also @aasthakourav20

**Comment by aasthakourav20** [comment]
> What input options will you give, will you consider Country selection also @aasthakourav20

It would include city & month(year) as input

**Comment by aasthakourav20** [comment]
> What input options will you give, will you consider Country selection also @aasthakourav20

Hey , will you assign this to me ??

**Comment by Rakesh9100** [comment]
How will you determine that particular city moon phase, will you use API for that @aasthakourav20
